sc/shared/src/main/scala/scalan/staged/Transforming.scala

```diff
@@ -65,10 +65,6 @@ trait Transforming { self: Scalan =>
   def beginPass(pass: Pass): Unit = {
     _currentPass = pass
   }
-  /** Called to let this IR context to finalized the given pass. */
-  def endPass(pass: Pass): Unit = {
-    _currentPass = Pass.defaultPass
-  }
```

Comment:
This methods complement each other. Either both can be removed or left as is.

Reply:
If you mean beginPass / endPass , then the former one is used while the latter is not.

Reply:
I would leave it for consistency, but it may as well be removed.